lib/run/summary.js

```diff
@@ -319,6 +319,9 @@ _.assign(RunSummary, {
             timings = timings && timings.timings;
             timingPhases = timings && sdk.Response.timingPhases(timings);
 
+            summary.run.timingPhases = timingPhases;
```

Comment:
So the fields for run are given to be stats, timings, executions, failures, transfers and error. For request it would be timingPhases, executions, transfers, failures and error, right?